templates/repo/issue/list.tmpl

```diff
@@ -220,7 +220,12 @@
 
 					<p class="desc">
 						{{ $timeStr := TimeSinceUnix .GetLastEventTimestamp $.Lang }}
-						{{$.i18n.Tr .GetLastEventLabel $timeStr .Poster.HomeLink .Poster.Name | Safe}}
+
+						{{if gt .Poster.ID 0}}
+							{{$.i18n.Tr "repo.issues.opened_by" $timeStr .Poster.HomeLink .Poster.Name | Safe}}
```

Comment:
This changes existing behavior, why not `.GetLastEventLabel`?

Reply:
My bad, remediating.

Reply:
Done.